core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -49,6 +49,20 @@ oppia.factory('PlaythroughService', [
     var cycleIdentifier = {};
     var visitedStates = [];
 
+    /**
+     * A quit action is recorded every time a MaybeQuit event is encountered.
+     * This function extracts the old quit action out of the playthrough
+     * actions list.
+     */
+    var extractOldQuitAction = function() {
```

Comment:
Extract implies that you are returning the old actions somehow. Maybe rename to `removeQuitActions` instead?

Reply:
Done.